framework/build.mk

```diff
@@ -240,6 +245,21 @@ ifneq (,$(findstring g95,$(mpif77_command)))
 	libmesh_FFLAGS += -r8
 endif
 
+#ifort
+ifneq (,$(findstring ifort,$(mpif90_command)))
+	libmesh_FFLAGS += -nofpp
```

Comment:
Why no preprocessor?

Reply:
I copied this one from my app, but I guess it is not needed.

Reply:
Yeah - I would say remove it unless we have a specific reason.

I think the bigger concern is the gfortran ones which may be causing BISON to fail.